components/works/work.tsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { ArrowRight } from 'react-feather';
+import placeholderImg from '../../public/images/placeholder.jpg';
+import { cn } from '../../utils/cn';
+import { IContent } from '../../utils/content';
+
+// TODO: use some type of hash function to generate the colors
+const colorsLookup = {
+  'code-collab': 'bg-accent-delight text-white',
+  'health-track': 'bg-accent-8',
+  'task-buddy': 'bg-accent-bob text-white',
+};
+
+export function WorkItem({ work, grid }: { work: IContent; grid?: boolean }) {
+  if (grid) return <WorkGridItem work={work} />;
+
+  const { title, slug, description } = work;
+  return (
+    <Link
+      href={`/works/${slug}`}
+      className={cn(
+        'group rounded-xl select-none no-underline flex flex-col lg:py-10 w-full md:flex-row transition-transform hover:cursor-pointer hover:scale-[102%] active:scale-95',
+        colorsLookup[slug as keyof typeof colorsLookup],
```

Comment:
Added a type to avoid `any`. `slug` is cast to the type `keyof typeof colorsLookup`, telling TS that the `slug` is a key of the `colorsLookup` object.